### PR TITLE
Use .netrc to do github pushes

### PR DESCRIPTION
**Pull request by vtemian**
Let drone publish the patched Wordpress, so we don't need to manually run the make scripts.

We do that by using the `.netrc` config via the `setup-github-credentials.sh` script, that is inspired by the `kluster-toolbox` way of doing this kind of operations.

##### Tests:
- make a new branch from `fix-drone`
- remove 
```
when:
      event: push
      branch: master
```

- add a dummy commit
- push
- it **should** start a new Drone job that will succeed and will push new commits to https://github.com/presslabs/wordpress

### CODE REVIEW

File: setup-github-credentials.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```
Comment (calind) [inline setup-github-credentials.sh]: This should be moved into bfc.

File: .drone.yml [hunk 2]
```diff
@@ -4,11 +4,18 @@ pipeline:
     pull: true
     environment:
       - ORIGIN_REPO=https://github.com/presslabs/wordpress.git
+      - GH_USER=presslabs-bot
+      - GH_EMAIL=bot@presslabs.com
+    secrets: [gh_token]
     commands:
       - apt-get update
```
Comment (calind) [inline .drone.yml]: Bfc already installs PHP, no need to install it here anymore.

File: setup-github-credentials.sh [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+fi
+
+if [ -z "$GH_TOKEN" ]; then
+    echo "You must define \"GH_PASSWORD\" environment variable" >&2
+    exit 2
+fi
+
+git config --global user.email ${GH_EMAIL:-no-reply@presslabs.com}
+git config --global user.name $GH_USER
+cat <<EOF > ~/.netrc
```
Comment (calind) [inline setup-github-credentials.sh]: We should append to `~/.netrc`

**Review by calind — CHANGES_REQUESTED** [review]

File: setup-github-credentials.sh (hunk 1 above)
Comment (vtemian) [reply setup-github-credentials.sh]: Moved via https://github.com/presslabs/bfc/pull/3

**Review by calind — APPROVED** [review]

**Pull request merged**